pkg/signal/signal_test.go

```diff
@@ -77,16 +76,14 @@ func (t *signalSuite) TestSignalSet(c *C) {
 	c.Assert(err, ErrorMatches, "signal number not supported: 16")
 	c.Assert(sm.isMuted(), Equals, true)
 	c.Assert(sm.isSignalMuted(SignalNatFillUp), Equals, true)
-	c.Assert(sm.isSignalMuted(SignalCTFillUp), Equals, true)
 	c.Assert(sm.isSignalMuted(SignalAuthRequired), Equals, true)
 
 	// 2 active signals
-	err = sm.UnmuteSignals(SignalNatFillUp, SignalCTFillUp)
+	err = sm.UnmuteSignals(SignalNatFillUp, SignalAuthRequired)
 	c.Assert(err, IsNil)
 	c.Assert(sm.isMuted(), Equals, false)
 	c.Assert(sm.isSignalMuted(SignalNatFillUp), Equals, false)
-	c.Assert(sm.isSignalMuted(SignalCTFillUp), Equals, false)
-	c.Assert(sm.isSignalMuted(SignalAuthRequired), Equals, true)
```

Comment:
Nit: We're losing this assert here, how about using another signal?

Same comment at a few other locations below

Reply:
There's no other signal left to test  :grin:. This is the best I could come up with on a quick look.

Reply:
OK fair enough :)